translator.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+"""Translates any text document into an .fb file
+
+First CLI argument should be a text file to be converted.
+Print to `.fb` file extension with ` > output_file.fb`
+
+Example:
+
+./translator.py original_text_file.txt > new_fb_file.fb
+
+Check translation with ./code_runner.py new_fb_file.fb"""
+
+import sys
+
+
+class Translator():
+    """Prepares common FizzBuzzLang functions and retrieves file text content
+
+    ---
+    Attributes:
+        look_up: dict -- self-populating dictionary of finary refs
```

Comment:
Unless you intend for these attributes to be a public interface, I would suggest putting these comments directly within the init method, next to the the definitions.
And if you do want these to be inputted as an interface, I would suggest explicitly making them kwargs of init

Reply:
Also, is `finary` a thing? Because I love it and think it should be one!

Reply:
I don't think there is any reason for them to be a public interface, no. I had looked at Google's docstring style guide and was not aware the class docstring should not include private attributes. I shall look at re-organising the comments. Thanks for the heads up!

`finary` is completely made up, but with the structure of the language, seemed the most fitting naming convention for the dictionary of characters 😁